src/Symfony/Component/Process/Contracts/CommandExecutorInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Process\Contracts;
+
+/**
+ * Command executor interface.
+ *
+ * @author Pululu Kinanga Andre <pululuandre@gmail.com>
+ */
+interface CommandExecutorInterface
+{
+    public function execute(string $command): ?string;
```

Comment:
This interface should not exist. It is a worse API than the whole process component (worse because it is a lot easier to be safe with the array-based API of the Process class than to pass a command line as a string where the caller is responsible for getting the escaping right)